libraries/AP_Airspeed/AP_Airspeed_MS5525.h

```diff
@@ -30,7 +30,13 @@
 class AP_Airspeed_MS5525 : public AP_Airspeed_Backend
 {
 public:
-    AP_Airspeed_MS5525(AP_Airspeed &frontend);
+    enum MS5525_ADDR {
+        MS5525_ADDR_0    = 0,
+        MS5525_ADDR_1    = 1,
```

Comment:
Can't this be ADDR_1/2? The only reason is to match the existing *MS5525D0_I2C_ADDR_1, MS5525D0_I2C_ADDR_2*

Reply:
Addressed.